cli/cmd/profile.go

```diff
@@ -3,12 +3,21 @@ package cmd
 import (
 	"bytes"
 	"errors"
+	"fmt"
 	"io"
+	"io/ioutil"
+	"net/http"
 	"os"
+	"regexp"
+	"sort"
 	"text/template"
 
+	"github.com/ghodss/yaml"
+	"github.com/go-openapi/spec"
```

Comment:
Looks like this library only supports OpenAPI v2. Maybe the CLI should abort with an error if you pass it a v3 spec?

Reply:
Hmmm... yeah the readme says v2 but all my testing has been with v3 specs.  🤷‍♂️